gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -31,7 +31,7 @@
  */
 public interface ResourceManager extends Service<ResourceManagerOptions> {
 
-  public static final String DEFAULT_CONTENT_TYPE = "application/octet-stream";
+  String DEFAULT_CONTENT_TYPE = "application/octet-stream";
```

Comment:
I might be missing some background here but why did this become an instance variable?